Makefile

```
@@ -61,6 +61,7 @@ OPERATOR_SDK_VERSION := $(eval OPERATOR_SDK_VERSION := $(shell cat versions/OPER
 KIND_VERSION         ?= v0.10.0
 HELM_VERSION         ?= v3.5.2
 HELM_DOCS_VERSION    ?= 1.5.0
+YQ_VERSION           ?= v4.6.2
```

sorry for reviewing late, it seems new version for YQ v4.6.3

---

thanks for reviewing. 
```suggestion
YQ_VERSION           ?= v4.6.3
```